easybuild/easyconfigs/g/GDAL/GDAL-3.10.0-foss-2024a.eb

```diff
@@ -0,0 +1,81 @@
+easyblock = 'CMakeMake'
+
+name = 'GDAL'
+version = '3.10.0'
+
+homepage = 'https://www.gdal.org'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://download.osgeo.org/%(namelower)s/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['%(name)s-3.6.2_fix-python-CC-CXX.patch']
+checksums = [
+    {'gdal-3.10.0.tar.xz': 'af821a3bcf68cf085724c21c9b53605fd451d83af3c8854d8bf194638eb734a8'},
+    {'GDAL-3.6.2_fix-python-CC-CXX.patch': '859b874b0c8ff7626a76d51f008bf05b7f89a35b325bdd1d126d2364154acc63'},
+]
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('pkgconf', '2.2.0'),
+    ('Bison', '3.8.2'),
+]
+dependencies = [
+    ('Python', '3.12.3'),
+    ('netCDF', '4.9.2'),
+    ('expat', '2.6.2'),
+    ('GEOS', '3.12.2'),
+    ('SQLite', '3.45.3'),
+    ('libarchive', '3.7.4'),
+    ('libxml2', '2.12.7'),
+    ('libpng', '1.6.43'),
+    ('libjpeg-turbo', '3.0.1'),
+    ('LibTIFF', '4.6.0'),
+    ('zlib', '1.3.1'),
+    ('cURL', '8.7.1'),
+    ('PCRE', '8.45'),
+    ('PROJ', '9.4.1'),
+    ('libgeotiff', '1.7.3'),
+    ('SciPy-bundle', '2024.05'),
+    ('HDF5', '1.14.5'),
+    ('HDF', '4.3.0'),
+    ('Armadillo', '14.0.3'),
+    ('CFITSIO', '4.4.1'),
+    ('zstd', '1.5.6'),
+    ('giflib', '5.2.2'),
```

Comment:
can we use v5.2.1 here instead? is already used as dep for
>{'ASE-3.23.0-gfbf-2024a.eb', 'matplotlib-3.9.2-gfbf-2024a.eb', 'modin-0.32.0-foss-2024a.eb', 'GPAW-24.6.0-foss-2024a.eb', 'Seaborn-0.13.2-gfbf-2024a.eb', 'dask-2024.9.1-gfbf-2024a.eb', 'bokeh-3.6.0-gfbf-2024a.eb', 'nf-core-2.14.1-foss-2024a.eb'}

Reply:
3ed0dcf